lib/screens/home/local_widget/province_card.dart

```diff
@@ -0,0 +1,238 @@
+import 'package:cambodia_geography/configs/route_config.dart';
+import 'package:cambodia_geography/constants/config_constant.dart';
+import 'package:cambodia_geography/helper/number_helper.dart';
+import 'package:cambodia_geography/models/tb_district_model.dart';
+import 'package:cambodia_geography/models/tb_province_model.dart';
+import 'package:flutter/material.dart';
+import 'package:scroll_to_index/scroll_to_index.dart';
+
+class ProvinceCard extends StatelessWidget {
+  const ProvinceCard({
+    Key? key,
+    required this.index,
+    required this.tabController,
+    required this.scrollController,
+    required this.province,
+    required this.district,
+  }) : super(key: key);
+
+  final int index;
+  final TabController tabController;
+  final AutoScrollController scrollController;
+  final TbProvinceModel province;
+  final List<TbDistrictModel> district;
+
+  @override
+  Widget build(BuildContext context) {
+    return AutoScrollTag(
```

Comment:
you can use AutoScrollTag in the home directly, so you may not have to paste index & `tabController` or `scrollController`  here & all animation relate sticky headers are in one place (home screen) which easier to read. Just like in `food_panda_sticky_header`.